howtos/custom_generators.rst

```diff
@@ -151,21 +85,12 @@ Then, write in it the following **conanfile.py**:
         url = "https://github.com/memsharded/conan-premake"
         license = "MIT"
 
-        def build(self):
-            pass
-
-        def package_info(self):
-            self.cpp_info.includedirs = []
-            self.cpp_info.libdirs = []
-            self.cpp_info.bindirs = []
+This is a full working example. Note the ``PremakeDeps`` class as a helper. The generator is creating Premake information for each
+individual library separately, then also an aggregated information for all dependencies. This ``PremakeDeps`` wraps a single item of such
+information.
 
-This is a full working example. Note the ``PremakeDeps`` class as a helper. The generator is
-creating premake information for each individual library separately, then also an aggregated
-information for all dependencies. This ``PremakeDeps`` wraps a single item of such information.
-
-Note the **name of the package** will be **PremakeGen/0.1@user/channel** as that is the name given
-to it, while the generator name is **Premake**. You can give the package any name you want, even
-matching the generator name if desired.
+Note the **name of the package** will be **PremakeGen/0.1@user/channel** as that is the name given to it, while the generator name is
+**Premake**. You can give the package any name you want, even matching the generator name if desired.
```

Comment:
Is it case sensitive? Can I give the generator a custom name?

Reply:
It is a package name, no further restriction applied. And the generator name is explained in the reference. I have added a clarification here too